### DISCUSSION TITLE
modify  Divider component in Autocomplete component

### DISCUSSION THREAD

**Discussion by MR-Mostafa** [mantinedev/mantine#1604] [Feature requests]
If the data sent to `Autocomplete` has a `group` property, this component will group the data.
It will also use the `Divider` component for grouping the data.
Is there any way to modify this component? (I mean the `Divider` component, for grouping data)

**Comment by rtivital** [comment]
You can target divider with static class – https://codesandbox.io/s/currying-smoke-y4sxd5?file=/src/App.tsx

**Comment by MR-Mostafa** [comment]
I meant other changes than style.

**Reply by rtivital** [reply]
What other changes?

**Reply by MR-Mostafa** [reply]
For example, I want to add a button next to the `Divider` element that clicks on it to perform a specific operation.

**Reply by rtivital** [reply]
No, that will be an accessibility violation, we will not add such feature

**Reply by rtivital** [reply]
You can use custom item component to implement such logic
